formula-1.js

```diff
@@ -32,4 +32,37 @@ console.log(drivers);
 ******************************************************************************************/
 
 
-// Write code here
\ No newline at end of file
+// Write code here
+
+function Driver(player,team,country) {
+  this.player = player;
+  this.team = team;
+  this.country = country;
+}
+const lewisHamilton = new Driver('Lewis Hamilton', 'Mercedes', 'United Kingdom')
+const valtteriBottas = new Driver('Valtteri Bottas', 'Mercedes', 'Finland')
+const maxVerstappen = new Driver('Max Verstappen', 'Red Bull Racing', 'United Kingdom')
+const danielRicciardo = new Driver('Daniel Ricciardo', 'Renault', 'Australia')
+const charlesLeclerc = new Driver('Charles Leclerc', 'Ferrari', 'Finland')
+const sergioPerez = new Driver('Sergio Perez', 'Racing Point', 'Mexico')
+const landoNorris = new Driver('Lando Norris', 'McLaren', 'United Kingdom')
+const alexanderAlbon = new Driver('Alexander Albon', 'Red Bull Racing', 'Thailand')
+const pierreGasly = new Driver('Pierre Gasly', 'AlphaTauri', 'France')
+const carlosSainz = new Driver('Carlos Sainz', 'McLaren', 'Spain')
+const lanceStroll = new Driver('Lance Stroll', 'Mercedes', 'Canada')
+const estebanOcon = new Driver('Esteban Ocon', 'Renault', 'France')
+const sebastianVettel = new Driver('Sebastian Vettel', 'Ferrari', 'Germany')
+const daniilKvyat = new Driver('Daniil Kvyat', 'AlphaTauri', 'Russian Federation')
+const antonioGiovinazzi = new Driver('Antonio Giovinazzi', 'Alfa Romeo', 'Italy')
+const kimiRäikkönen = new Driver('Kimi Räikkönen', 'Alfa Romeo', 'Finland')
+const romainGrosjean = new Driver('Romain Grosjean', 'Haas', 'France')
+const kevinMagnussen = new Driver('Kevin Magnussen', 'Haas', 'Denmark')
+const nicholasLatifi = new Driver('Nicholas Latifi', 'Williams', 'Canada')
+const georgeRussell = new Driver('George Russell', 'Williams', 'United Kingdom')
+
+for (let i = 0; i < drivers.length; i++) {
+  let driverObject = drivers[i];
+  let {player,team,country} = driverObject;
+  const eachDriver = new Driver(player,team,country);
+  console.log(eachDriver);
+}
```

Comment:
This is good– although you could use a `for...of loop` here– but instead of `console.log(eachDriver);` you could push `eachDriver` into a new array. Alternatively, you could us `Array.prototype.map()` e.g. `drivers.map(driver => new Driver(...Object.values(driver));`